packages/synthetic-chain/package.json

```diff
@@ -1,8 +1,10 @@
 {
   "name": "@agoric/synthetic-chain",
-  "version": "0.0.8",
+  "version": "0.0.9",
   "description": "Utilities to build a chain and test proposals atop it",
-  "bin": "dist/cli/cli.js",
+  "bin": {
+    "synthetic-chain": "dist/cli/cli.js"
+  },
```

Comment:
wasn't there some issue with this we reverted this change before?

Reply:
I don't know whether it was intentional. Do you recall something? Regardless, the `npm` command changes it while publishing so I think this is actually what's happening